examples/Building-and-deploying-multi-stage-RecSys/01-Building-Recommender-Systems-with-Merlin.ipynb

```diff
@@ -7,7 +7,7 @@
    "metadata": {},
```

Comment:
<p>may be we can add some text right above this cell to explain what we are doing here and what we are registering to feast repo. I can do that.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/1022/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>